src/index.json

```diff
@@ -56400,6 +56400,51 @@
                 "sha256Digest": "58074ff73a72a1a9c76d406a4b6988a7b714dfaa9259b92172a7b6f66680db31"
             }
         ],
+        "stack-hci-vm": [
+            {
+                "downloadUrl": "https://hybridaksstorage.z13.web.core.windows.net/SelfServiceVM/CLI/stack_hci_vm-0.1.2-py3-none-any.whl",
+                "filename": "stack_hci_vm-0.1.2-py3-none-any.whl",
+                "metadata": {
+                    "azext.isExperimental": true,
+                    "azext.minCliCoreVersion": "2.15.0",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
```

Comment:
BTW the python version which less than 3.8 is not supported. Please update it as well

Reply:
Should I keep the line without any version of Python specified?